docker/Dockerfile

```diff
@@ -9,8 +9,11 @@ RUN chmod 755 /bin/console && \
     apt-get install -y circonus-field-broker && \
     apt-get install -y --allow-unauthenticated circonus-field-broker-crashreporter && \
     apt-get install -y telnet && \
+    mkdir /opt/napp/package && \
+    /opt/circonus/bin/curl -o /opt/napp/package/circonus-field-broker-core-stock.deb http://pilot.circonus.net/ubuntu/pool/main/c/circonus-field-broker-core/`dpkg -l circonus-field-broker-core | awk '/broker-core/{print $2"-"$3"xenial_"$4".deb"}'` && \
```

Comment:
We might want to encode the LSB release codename here too, thinking ahead to 20.04/focal support.

Reply:
I don't think that necessary.  This stuff only makes sense for config file changes, which are only depend on version (not release, or platform, or even architecture)